### PR TITLE
Feat/account migration

### PR DESCRIPTION
**Pull request by tomicvladan**

### CODE REVIEW

File: src/pages/migrate/migrate.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        {
+          user_name: newUsername,
+          password,
+        },
+        { withCredentials: true }
+      );
+
+      setStep(Steps.Complete);
+    } catch (error) {
+      console.error(error);
```
[the remark this answers is not in the document]
Comment (tomicvladan) [reply src/pages/migrate/migrate.tsx:64]: Good point. I've added additional check for insufficient funds and an error message for that case. But also updated the generic message to include message returned by server.

[the remark this answers is not in the document]
Comment (asabya) [reply src/pages/migrate/migrate.tsx:64]: Can we add a way to display user address?

[the remark this answers is not in the document]
Comment (tomicvladan) [reply src/pages/migrate/migrate.tsx:64]: We can, but we need another request to fetch address. Which endpoint does return address?

[the remark this answers is not in the document]
Comment (asabya) [reply src/pages/migrate/migrate.tsx:64]: If you are logged in, POST on `v1/user/export` with cookie. 

Response will look like this
```
{
    "user_name": "<USER>",
    "address": "<ADDRESS>"
}
```

**Review by molekilla — APPROVED** [review]
Looks good

File: src/pages/migrate/migrate.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (tomicvladan) [reply src/pages/migrate/migrate.tsx:64]: That's response for registration. For login I'm getting only this response: `{"message":"user logged-in successfully","code":200}`

**Review by asabya — APPROVED** [review]
LGTM

File: .default.env [hunk 2]
```diff
@@ -1,6 +1,7 @@
 REACT_APP_BEE_URL=http://localhost:1633
 REACT_APP_BEE_DEBUG_URL=http://localhost:1635
 REACT_APP_RPC_URL=http://localhost:9545
+REACT_APP_FAIROS_URL=http://localhost:9090
```
Comment (tfius) [inline .default.env:4]: please add to readme 
1. link to fairos repo 
2. link to installation fairos instructions 
3. specify why it is required

**Review by tfius — APPROVED** [review]

File: .default.env (hunk 2 above)
Comment (tfius) [reply .default.env:4]: add run fdp-play with another command: 'fdp-play start --fresh --blockchain-image fairdatasociety/fdp-contracts-blockchain:1.1.1'

**Pull request merged**